src/main/java/by/exadel/internship/entity/Form.java

```diff
@@ -82,4 +82,7 @@
     @JoinColumn(name = "fm_id")
     private List<TimeForCall> timeForCallList;
 
+    @Column(name = "fm_inship_id")
+    private UUID internship;
```

Comment:
may be better rename it to internshipId as long as it is not an internship entity

Reply:
done